pkg/blockstorage/azure/environments.go

```diff
@@ -0,0 +1,83 @@
+package azure
```

Comment:
I have a general question for this file. IIUC, this is meant to replace `go/pkg/mod/github.com/azure/go-autorest/autorest@v0.11.29/azure/environments.go` right? Was there any official recommendation on how to replace it? It seems like they pulled the rug from under our feet here if we need to manually replace this file.

Reply:
Oh wow. Do they have any recommendation how how to replace it?

Reply:
No recommendations.